api/DeprecationReportBody.json

```diff
@@ -11,7 +11,16 @@
           "chrome_android": "mirror",
           "edge": "mirror",
           "firefox": {
-            "version_added": false
+            "version_added": "65",
+            "flags": [
+              {
+                "type": "preference",
+                "name": "dom.reporting.enabled",
+                "value_to_set": "true"
+              }
+            ],
+            "partial_implementation": true,
+            "notes": "Not supported in workers"
```

Comment:
FF Current IDL says only supported in Window, while the spec indicates worker required. There is no test that says whether chrome  supports it or not so there is no "worker support" feature. So what I have done is marked as partial and put a note about worker support.